File: Server/src/main/java/jk_5/nailed/server/command/CommandStartGame.java
package jk_5.nailed.server.command;

import net.minecraft.command.*;

import jk_5.nailed.api.map.*;

/**
 * No description given
 *
 * @author jk-5
 */
public class CommandStartGame extends NailedCommand {

    public CommandStartGame() {
        super("startgame");
    }

    @Override
    public void processCommandWithMap(ICommandSender sender, Map map, String[] args) {
        map.getGameManager().startGame();
    }
}

File: Server/src/main/scala/jk_5/nailed/server/command/CommandStartGame.scala
package jk_5.nailed.server.command

import net.minecraft.command.ICommandSender
import jk_5.nailed.api.map.Map

/**
 * No description given
 *
 * @author jk-5
 */
object CommandStartGame extends ScalaCommand {

  val name = "startgame"
  val usage = "/startgame - Starts the game in your current map"

  override def processCommandWithMap(sender: ICommandSender, map: Map, args: Array[String]){
    map.getGameManager.startGame()
  }
}
